java/src/lib.rs

```diff
@@ -195,9 +182,22 @@ pub extern "system" fn Java_glide_ffi_resolvers_ScriptResolver_storeScript<'loca
     _class: JClass<'local>,
     code: JString,
 ) -> JObject<'local> {
-    let code_str: String = env.get_string(&code).unwrap().into();
-    let hash = glide_core::scripts_container::add_script(&code_str);
-    JObject::from(env.new_string(hash).unwrap())
+    handle_panics(
+        move || {
+            fn store_script<'a>(
```

Comment:
In theory, `add_script` can `panic`, will this cause a stack overflow?

Reply:
There should not be a stack overflow. I do not use recursive calls to handle_panics. The `catch_unwind` will convert the panic into a Java Exception and throw it, which is similar to the panic handling test case shown in `ffi_test.rs`.